### PR TITLE
Adding client reinterpretation lag technique

### PR DESCRIPTION
**Pull request by SamuelBellomo**
**Select the type of change:**

- [ ] Small Changes - Typos, formatting, slight revisions
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site and Docusaurus

STILL NEEDS MORE DETAILS

**Purpose of the Pull Request:**

**Issue Number:**

### CODE REVIEW

File: docs/learn/dealing-with-latency.md [hunk 1]
```diff
@@ -279,6 +279,11 @@ Advanced games will have most of their world predicted, allowing the client and
 There's no prediction implementation right now in Netcode for GameObjects, but you can implement your own. See our [roadmap](https://unity.com/roadmap/unity-platform/multiplayer-networking) for more information.
 :::
 
+### Client Side Reinterpretation (state solving)
+For a few reasons, clients can't always stay in sync tick by tick, frame by frame. It's possible to adapt your gameplay to allow your clients to be desynced for a few seconds, as long as they sync back again eventually.
+This is usually the case for death ragdolls in games where a dead character's body doesn't influence your gameplay anymore and so can be desynced from one client the another. The server will sync to all clients "this character is now dead" and will send it's death position. From then on, clients can apply all the ragdol physics they want with no need of syncing it between clients (as it has no gameplay impact, we don't care if I see the ragdol in a tree and you see the ragdol in a ditch).
```
Comment (11BelowStudio) [inline docs/learn/dealing-with-latency.md:284]: *ragdoll (not ragdol)

Comment (SamuelBellomo) [reply docs/learn/dealing-with-latency.md:284]: ah woops, thanks for the catch. This is mostly a draft, more polishing and details to come.

**Comment by s-omeilia-unity** [comment]
@SamuelBellomo What's going on with this PR? Is it still valid or should it be closed?

**Comment by SamuelBellomo** [comment]
> @SamuelBellomo What's going on with this PR? Is it still valid or should it be closed?

@s-omeilia-unity should be, just never got time to come back to this one. It's pretty old, I'll try and see if I can reserve an hour or two this or next week to add more details.

**Comment by s-omeilia-unity** [comment]
> @s-omeilia-unity finally got some time to come back to this. I feel like "Client Reinterpretation" could be confused too easily with Client Prediction in conversations, I looked for a name that was more unique. "Controlled Desync" still shows the important part of the technique, where we allow clients to be desynced as long as they converge to a common state. I'm open to other ideas though, let me know what you think. I'd like to wait to get netcode team's feedback on this before merging though. I'll share internally.

I agree with "Controlled Desync". It makes sense to me.

**Comment by unity-cla-assistant** [comment]
[CLA assistant check](https://cla-assistant.cds.internal.unity3d.com/Unity-Technologies/com.unity.multiplayer.docs?pullRequest=742) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.cds.internal.unity3d.com/Unity-Technologies/com.unity.multiplayer.docs?pullRequest=742) before we can accept your contribution.<br/><hr/>**Larah Armstrong** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.cds.internal.unity3d.com/check/Unity-Technologies/com.unity.multiplayer.docs?pullRequest=742) it.</sub>

**Pull request merged**